specification/quantumleap.yml

```diff
@@ -696,7 +695,8 @@ paths:
               {
                 "data": {
                   "entityId": "Kitchen1",
-                  "index": ["1510826981388", "1510826990438", "1510826999725"],
+                  "index": ["2018-01-05T15:44:34", "2018-01-06T15:44:59",
```

Comment:
shouldn't have also the time zone reference?

Reply:
Good point. We stated in the docs that the returned time is always in UTC.
However, it may be nice to give this info back as well, wdyt? 

Also, adding some tests (see https://github.com/smartsdk/ngsi-timeseries-api/pull/95/commits/47356901b06a550b1f1ac7cdeb86abcd99cfbeb1) I've found an inconsistency, so I've adjusted #97 to cover these issues in a new PR.